R/comparePlot.R

```diff
@@ -107,23 +91,19 @@
 #'
 #' @export
 comparePlot <- function(compareDF,
+                        plotType = "canvasXpress",
                         pThreshold = 0.01,
                         xlab = NULL, ylab = NULL,
                         title = NULL,
-                        symbolSize = c(4, 4, 4, 2),
+                        symbolSize = c(10, 10, 10, 4),
                         symbolShape = c(21, 21, 21, 20),
                         symbolColor = c("black", "grey0", "grey1", "grey25"),
```

Comment:
How you determined color values?

Reply:
If you mean rgb values we uses function col2rgb() and if you mean acceptable options then you can go through cx api . But more or less all colors are acceptable.